mcs/class/System/System.Net.Sockets/Socket.cs

```diff
@@ -1147,6 +1147,13 @@ public void Bind (EndPoint local_end)
 
 			if (local_end == null)
 				throw new ArgumentNullException("local_end");
+				
+#if NET_4_5
```

Comment:
Good to know! This was modeled after the recent additions of the MapToIPV4/6 methods which were ifdefed. Will remove them!